cypress/fixtures/constants.ts

```diff
@@ -318,6 +318,9 @@ export class Constants {
             coreImportMessage: 'The content below is imported from /core',
             otherImportMessage: 'The content below is imported from /other'
         },
+        umdFederationHeader1: 'Host App 1',
+        umdFederationHeader2: 'MF App 01',
+        umdFederationHeader3: 'UMD App2'
```

Comment:
You can make one object for these constants.
Something like that:
```
umdFederation: {
   App1: {
      firstHeader:  'Host App 1',
      secondHeader:  'MF App 01',
      thirdHeader:  'UMD App2'
   }
}
```

And then use them in the test 
```
Constants.elementText.umdFederation.App1.firstHeader
Constants.elementText.umdFederation.App1.secondHeader
...
```